settings.gradle

```
@@ -1,3 +1,4 @@
 rootProject.name = 'otus_java_professional'
 include 'hw01-gradle'
-include 'hw02-arraylist'
\ No newline at end of file
+include 'hw02-arraylist'
+include 'hw03-annotations'
```

Папка у вас называется `hw03-annotaions`
```suggestion
include 'hw03-annotaions'
```

Но, скорее всего, надо папку переименовать, а не здесь.